src/lib/crypto_params/crypto_params.ml

```diff
@@ -6,7 +6,10 @@ module Tock = struct
   module Full = Pickles.Impls.Wrap
   module Run = Pickles.Impls.Wrap
 
-  let group_map_params () = Lazy.force Group_map_params.params
+  let group_map_params =
```

Comment:
This should probably still be a function to avoid doing work immediately at startup

Reply:
I just benchmarked all of these generator files using the same technique:

```ocaml
let main = 
  let start = Time.now () in
  (* body of main function *)
    let e = Time.now () in
  let () = print_endline (Float.to_string @@ Time.Span.to_ms (Time.diff e start)) in
  exit 0
  ```
  
  each PR's description contains the benchmark, and this one takes less than `.2ms` to compute. I would argue that this kind of premature optimization isn't necessary. Also assuming the `lazy` semantics of rust, if you want the function to stay I feel like we should keep the value listed as `lazy` no?